runtimes/pythonrt/runner/tests/values_test.py

```diff
@@ -0,0 +1,143 @@
+import pb.autokitteh.values.v1.values_pb2 as pb
+from values import wrap, unwrap
+from collections import namedtuple
+from datetime import datetime, timedelta, UTC
+import pytest
+import google.protobuf.timestamp_pb2 as timestamp_pb2
+import google.protobuf.duration_pb2 as duration_pb2
+
+
+def check(u, **kwargs):
+    v = pb.Value(**kwargs)
+    assert wrap(u) == v
+    assert unwrap(v) == u
+
+
+def intv(n):
+    return pb.Value(integer=pb.Integer(v=n))
+
+
+@pytest.mark.parametrize(
+    "u,v",
+    [
```

Comment:
I prefer you place the list of tests as a variable and use it in the decorator:

```python
wrap_cases = [ 
.... 
]

@pytest.mark.parametrize("u,v", wrap_cases)
def test_wrap(u, v):
    ...
```

Reply:
done in a new pr